frontend/src/main/java/com/google/tripmeout/frontend/PlaceVisitModel.java

```diff
@@ -0,0 +1,28 @@
+package com.google.tripmeout.frontend;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class PlaceVisitModel {
+  public static Builder builder() {
+    return new AutoValue_PlaceVisitModel.Builder();
+  }
+
+  public abstract String placeId();
+  public abstract String name();
+  public abstract String tripId();
+  public abstract String userMark(); // must-see, if-time, don't-care, null
+  public abstract float latitude();
```

Comment:
Do we have specific reason to store this as a `float`? For example, does the Maps API only handle coordinates with 32-bit precision anyway? If the answer is yes and we're only storing/passing through lat/long, this should be fine. However, if we're ever mathematically transforming the coordinates down the line, we'll want to store these as `double`s for increased precision.

Reply:
i looked into the java maps api and it does use doubles to represent latitude and longitude so I've changed it accordingly